Calculator/Calculator/Model/OperandFormatter.swift

```diff
@@ -10,61 +10,41 @@ import Foundation
 enum OperandFormatter {
     static func formatStringOperand(_ operand: String) -> String {
         let operandNumber = NSDecimalNumber(string: operand)
-        return formatNumberToString(operandNumber)
+        let numberFormatter = NumberFormatter()
+        numberFormatter.numberStyle = .decimal
+        numberFormatter.maximumFractionDigits = 11
+        numberFormatter.maximumIntegerDigits = 12
+        numberFormatter.roundingMode = .halfUp
+        
+        guard let numberFormatted = numberFormatter.string(for: operandNumber)
+        else {
+            return CalculatorNamespace.empty
+        }
+        
+        return numberFormatted
```

Comment:
```suggestion
        return numberFormatter.string(for: operandNumber) ?? CalculatorNamespace.empty
```

`guard문`의 변수를 여러 곳에서 쓰이지 않는 이상 `병합연산자`로 간결하게 표현해도 괜찮다는 생각이 듭니다!